spec/helpers/admin/claims_helper_spec.rb

```diff
@@ -0,0 +1,61 @@
+require "rails_helper"
+
+describe Admin::ClaimsHelper do
+  let(:school) { schools(:penistone_grammar_school) }
+
+  describe "eligibility_answers" do
+    let(:eligibility) do
+      build(
+        :student_loans_eligibility,
+        qts_award_year: "2013_2014",
+        claim_school: school,
+        current_school: school,
```

Comment:
To be safe, how about we assign two different schools to make extra sure the values being returned are the correct ones? We already saw a similar bug slip through in the CSV export.